src/pages/submit/tender.tsx

```diff
@@ -0,0 +1,422 @@
+import React, { useEffect, useMemo, useRef, useState } from 'react'
+import Helmet from 'react-helmet'
+import { Controller, SubmitHandler, useForm } from 'react-hook-form'
+
+import Label from 'decentraland-gatsby/dist/components/Form/Label'
+import MarkdownTextarea from 'decentraland-gatsby/dist/components/Form/MarkdownTextarea'
+import Head from 'decentraland-gatsby/dist/components/Head/Head'
+import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
+import Paragraph from 'decentraland-gatsby/dist/components/Text/Paragraph'
+import useAuthContext from 'decentraland-gatsby/dist/context/Auth/useAuthContext'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { navigate } from 'decentraland-gatsby/dist/plugins/intl'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { SelectField } from 'decentraland-ui/dist/components/SelectField/SelectField'
+import isEthereumAddress from 'validator/lib/isEthereumAddress'
+
+import { Governance } from '../../clients/Governance'
+import Field from '../../components/Common/Form/Field'
+import ErrorMessage from '../../components/Error/ErrorMessage'
+import MarkdownNotice from '../../components/Form/MarkdownNotice'
+import ContentLayout, { ContentSection } from '../../components/Layout/ContentLayout'
+import LoadingView from '../../components/Layout/LoadingView'
+import CoAuthors from '../../components/Proposal/Submit/CoAuthor/CoAuthors'
+import LogIn from '../../components/User/LogIn'
+import { SUBMISSION_THRESHOLD_PITCH } from '../../entities/Proposal/constants'
+import { NewProposalTender, newProposalTenderScheme } from '../../entities/Proposal/types'
+import usePreselectedProposal from '../../hooks/usePreselectedProposal'
+import useURLSearchParams from '../../hooks/useURLSearchParams'
+import useVotingPowerDistribution from '../../hooks/useVotingPowerDistribution'
+import loader from '../../modules/loader'
+import locations from '../../modules/locations'
+
+import './submit.css'
+
+const initialState: NewProposalTender = {
+  linked_proposal_id: '',
+  project_name: '',
+  summary: '',
+  problem_statement: '',
+  technical_specification: '',
+  use_cases: '',
+  deliverables: '',
+  target_release_date: new Date(),
+}
+const schema = newProposalTenderScheme.properties
+
+export default function SubmitTenderProposal() {
+  const t = useFormatMessage()
+  const params = useURLSearchParams()
+  const [account, accountState] = useAuthContext()
+  const accountBalance = isEthereumAddress(params.get('address') || '') ? params.get('address') : account
```

Comment:
I don't know why we are calling this accountBalance jaja 🙈. 
Can you also rename it in draft and governance submit pages please?

Reply:
Awful copy-pasting 🤣  I think the variable is not necessary, I'll check